src/makeRobot.js

```diff
@@ -38,7 +38,56 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+    get location() {
```

Comment:
```suggestion
    },
    
    get location() {
```
Оставляй пробелы между названиями свойства для читаемости